### ISSUE TITLE
The server does not work in background

### ISSUE THREAD

**Issue by nokai** [jxcore/jxcore-cordova#57]
I'm working with xcode and ios. (sample)

Works again when you return from background

Why is this happening?

Thanks

**Comment by ktrzeciaknubisa** [comment]
Do you mean the app is restarted?

**Comment by nokai** [comment]
No.

I mean, that while the app is running on background, the url (ip addresses 127.0.0.1:3000) does not work (Not found)

**Comment by nokai** [comment]
Nobody knows how the application will not pause in background ?

**Comment by obastemur** [comment]
iOS doesn't allow any (including native Objective C) thread is running when the app is on background. See https://developer.apple.com/library/ios/documentation/iPhone/Conceptual/iPhoneOSProgrammingGuide/BackgroundExecution/BackgroundExecution.html

**Comment by nokai** [comment]
But in the emulator run in background without problems, but in the device no works in background..

I do not understand . Why?

**Comment by obastemur** [comment]
IOS Simulator and real device do not function similarly. iOS does not support backround threads without a particular reason

**Comment by nokai** [comment]
Ok, but for example.

If I want the user to exchange files with the iPhone through the browser, I have to force the user to do nothing with your device, such as playing a game.

**Comment by obastemur** [comment]
Not exactly. Apple accepts some particular events as long as you set them as your intention. See the link and more I'd previously shared.

**Comment by ktrzeciaknubisa** [comment]
@nokai Also please remember, that putting the app into the background gives you no guarantee that the system will not kill it anytime it needs to release the resources etc. This kind of tasks as yours would need to to be explicitly defined as background task as the link above says, but I don't know if this is doable with JXcore at the moment.

Furthermore there are just few modes allowed for background tasks. Below is an excerpt from the link above:

> In iOS, only specific app types are allowed to run in the background:
> - Apps that play audible content to the user while in the background, such as a music player app
> - Apps that record audio content while in the background
> - Apps that keep users informed of their location at all times, such as a navigation app
> - Apps that support Voice over Internet Protocol (VoIP)
> - Apps that need to download and process new content regularly
> - Apps that receive regular updates from external accessories

The point is, that your task would not be allowed to do just anything and @obastemur just mentioned that in previous comment.

**Comment by nokai** [comment]
Ok, i want "Apps that need to download and process new content regularly"... 

Would it be possible do this with jxcore and Cordova? Where I can find a very easy example?

Thanks
